Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -30,6 +30,9 @@
 using System.Xml.Linq;
 using System.Xml.Serialization;
 using System.Web;
+using System.Text.RegularExpressions;
```

Comment:
do you need this?

Reply:
Removed that & other using statements that weren't used.